cmd/update.go

```diff
@@ -59,6 +59,7 @@ var (
 	githubAPIBase = "https://api.github.com"
 	download      = doDownload
 	osExecutable  = os.Executable
+	disableUpdateSubcommand = "false"
```

Comment:
Maybe just

```suggestion
	disableUpdateCmd = "false"
```

Also, are you allowed to use `-X` on lower-cased variables?

Reply:
Seems to work